script/Ciel/Day 14/14_01_CIEL14_1C.txt

```diff
@@ -616,7 +616,7 @@ She struck me square in the chest this time,
 [sha:211dd2395b7f980b5d9e0e90f918c337adf3919d]{
 -- Page 179, Offset 37542.
 -- 結果、俺は背中から地面に落下し、そして、
-As a result, I slam into the ground back-first, and then―――
+As a result, I slam into the ground back first, and then―――
```

Comment:
I think this should still have the dash, as in prev pr

Reply:
Without a dash is more common.
![image](https://user-images.githubusercontent.com/90473809/234772203-ba4dd67f-a64d-45dd-b1ed-e1676576cd86.png)

Reply:
Like all tools, you need to know its limits - if I search a newspaper corpus for 'back first', we get a lot of hits for contexts that _aren't_ the one we're using here:
> The results of the anemia work-up came back first.
> I've got to get the script back first.
> The only bad thing about writing is reading back first drafts.

These are going to skew the results. I think that back-first counts as an adjective here, so it gets compounded with a hyphen

Reply:
alright works for me